packages/manager/src/lib/format.ts

```diff
@@ -16,6 +16,8 @@ export const format = async (
 	filePath: string,
 	options?: FormatOptions,
 ): Promise<string> => {
+	const prettier = await import("prettier");
```

Comment:
Dynamic imports shouldn't significantly impact performance unless the module contains a lot of code. Prettier is large, so it's possible it could cause slow-downs, but I don't think it would be significant enough to worry about it.

Typically only the first load takes longer. I think Node.js's import cache speeds up subsequent imports.